### ISSUE TITLE
Performance time measurement

### ISSUE THREAD

**Issue by EsteveAguilera** [backpack-serializers/backpacker-kotlin#5]
More than an issue, this is a question I've got when looking at the performance test code.

Is there a good reason why you're logging the spend time inside the for loop?

```java
        long backpackToBytes = 0;
        for (int i = 0; i < LOOPS; i++) {
            long start = System.currentTimeMillis();
            new PersonBackpacker().toBytes(anna);
            backpackToBytes += (System.currentTimeMillis() - start); 
        }
```

Wouldn't it be "more accurate" to do it outside the for loop, and leaving inside the loop only the backpacker stuff?

```java
        long start = System.currentTimeMillis();
        for (int i = 0; i < LOOPS; i++) {
            new PersonBackpacker().toBytes(anna);
        }
        long backpackToBytes = (System.currentTimeMillis() - start);
```

**Comment by rocboronat** [comment]
Psss I think it's the same. Try it! Probably the results will be the same, but the code will be more readable 🙌🏻

**Comment by EsteveAguilera** [comment]
I'm not too much confident about the accuracy of the ```currentTimeMillis``` for a really short periods of time, and right now we're dealing wth it on every iteration.

Will do some tests and be back with some numbers 🏃

**Comment by rocboronat** [comment]
Oh! Now I got you, and you're totally right! You are my favorite #CBO!

**Comment by EsteveAguilera** [comment]
So, here're the numbers @rocboronat 

I've used the **compareUsingPerson** test and ran it 5 times.

|Inside loop|Outside loop|
|:-------------:|:-------------:|
|929ms|1055ms|
|1441ms|1103ms|
|1261ms|1082ms|
|1002ms|1016ms|
|1235ms|908ms|

It seems that there's not a huge difference, but tracking the time outside the loop gives more stable results.

More stable results plus more readable code sounds like a huge improvement!

**Comment by rocboronat** [comment]
Impressive! Great work! And thanks for sharing the results ❤️

**Comment by EsteveAguilera** [comment]
Fixed on #6
